### DISCUSSION TITLE
Why use a fork of python-snapcast?

### DISCUSSION THREAD

**Discussion by luar123** [music-assistant/support#2014] [Feature requests and ideas]
Hi,
the snapcast provider uses a custom fork of python-snapcast: snapcast-mod instead of the original snapcast package also used in home-assistant. Initially the Add/RemoveStream functionality was missing, but this has been added in the meantime together with several bug fixes.
I think it would benefit all projects if music-assistant would use the main python package as well and contribute new features and bug fixes there.

**Comment by OzGav** [comment]
@SantiagoSotoC

**Comment by SantiagoSotoC** [comment]
When I developed the provider, I tried to contribute to python-snapcast and was ignored. 
 
As those functions did not exist I created my own version

**Comment by marcelveldt** [comment]
FYI I switched to the regular version yesterday and it worked out of the box. So we're now based on the original package
